### PR TITLE
Update netfx and netcoreapp baseline files

### PR DESCRIPTION
**Pull request by weshaggard**
For netfx baseline files we are no including netstandard ref folder
in the list of resolve paths to find assemblies for netfx.

cc @ericstj

### CODE REVIEW

File: src/shims/ApiCompatBaseline.netcoreapp.netfx461.txt [hunk 1]
```diff
@@ -6,6 +6,12 @@ ApiCompat Error: 0 : Unable to resolve assembly 'Assembly(Name=System.Web.Applic
 ApiCompat Error: 0 : Unable to resolve assembly 'Assembly(Name=System.Configuration, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)' referenced by the contract assembly 'Assembly(Name=System.Web, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)'.
 ApiCompat Error: 0 : Unable to resolve assembly 'Assembly(Name=System.ServiceModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)' referenced by the contract assembly 'Assembly(Name=System.Xml.Serialization, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)'.
 Compat issues with assembly mscorlib:
+CannotRemoveBaseTypeOrInterface : Type 'System.Runtime.InteropServices.ImportedFromTypeLibAttribute' does not implement interface 'System.Runtime.InteropServices._Attribute' in the implementation but it does in the contract.
```
Comment (ericstj) [inline src/shims/ApiCompatBaseline.netcoreapp.netfx461.txt]: What are these about?

File: src/shims/ApiCompatBaseline.netfx.netstandard20.txt [hunk 2]
```diff
@@ -66,15 +64,6 @@ TypesMustExist : Type 'System.Reflection.MemberTypes' does not exist in the impl
 TypesMustExist : Type 'System.Reflection.ParameterModifier' does not exist in the implementation but it does exist in the contract.
```
Comment (ericstj) [inline src/shims/ApiCompatBaseline.netfx.netstandard20.txt]: These System.Reflection errors seem wrong.  For example: BindingFlags.

File: src/shims/ApiCompatBaseline.netcoreapp.netfx461.txt (hunk 1 above)
Comment (ViktorHofer) [inline src/shims/ApiCompatBaseline.netcoreapp.netfx461.txt]: We had a lot of these and put them into the ignore file. All Interfaces which are starting with an underscore aren't in netstandard/netcore.

**Comment by ViktorHofer** [comment]
@weshaggard Can you please do a rebase on upstream/master and merge my updated file with yours? If possible please put the new `CannotRemoveBaseTypeOrInterface` lines with "_Interfaces" to the .ignore file (at the right location, should be near the beginning).

File: src/shims/ApiCompatBaseline.netfx.netstandard20.txt (hunk 2 above)
Comment (weshaggard) [inline src/shims/ApiCompatBaseline.netfx.netstandard20.txt]: I didn't look at these in any deal I just refreshed from the build, but I agree those don't look right.

Comment (danmosemsft) [inline src/shims/ApiCompatBaseline.netfx.netstandard20.txt]: Do we have issues tracking clearing out hte rest of this baseline?

Comment (weshaggard) [inline src/shims/ApiCompatBaseline.netfx.netstandard20.txt]: I'm looking at the issues locally. I understand what is happening with these reflection types. I will look over the rest and make sure I understand it and if there is work to be done I will file issues.

File: src/shims/ApiCompatBaseline.uapaot.netstandard20.txt [hunk 3]
```diff
@@ -1,185 +1,4 @@
...
-CannotRemoveBaseTypeOrInterface : Type 'System.IO.Pipes.AnonymousPipeServerStream' does not inherit from base type 'System.MarshalByRefObject' in the implementation but it does in the contract.
-CannotRemoveBaseTypeOrInterface : Type 'System.IO.Pipes.NamedPipeClientStream' does not inherit from base type 'System.MarshalByRefObject' in the implementation but it does in the contract.
-CannotRemoveBaseTypeOrInterface : Type 'System.IO.Pipes.NamedPipeServerStream' does not inherit from base type 'System.MarshalByRefObject' in the implementation but it does in the contract.
-CannotRemoveBaseTypeOrInterface : Type 'System.IO.Pipes.PipeStream' does not inherit from base type 'System.MarshalByRefObject' in the implementation but it does in the contract.
-Compat issues with assembly System.IO.UnmanagedMemoryStream:
-CannotRemoveBaseTypeOrInterface : Type 'System.IO.UnmanagedMemoryStream' does not inherit from base type 'System.MarshalByRefObject' in the implementation but it does in the contract.
 Compat issues with assembly System.Memory:
 TypesMustExist : Type 'System.ReadOnlySpan<T>' does not exist in the implementation but it does exist in the contract.
```
Comment (danmosemsft) [inline src/shims/ApiCompatBaseline.uapaot.netstandard20.txt]: package authoring issue needed?

Comment (ericstj) [inline src/shims/ApiCompatBaseline.uapaot.netstandard20.txt]: No that's a library problem with the UAPAOT build of System.Memory.  Likely it is typeforwarding to CoreLib and it hasn't yet been implemented in the UAPAOT CoreLib.

**Comment by weshaggard** [comment]
I'm still working on this PR because I'm trying to get to the bottom of why I get different results when building with allconfigurations vs vertical builds. There is definitely some fishing things going on that I want to understand.

**Comment by weshaggard** [comment]
There are only 2 libraries which have issues in netfx.netstandard20 which are System.Security.Principal.Windows and System.Security.Cryptography.Cng and the issues they are complaining about are legitimate. They are interesting cases that we should think about a little more. They are both above netstandard2.0 but have netstandard refs which adds additional members that do not exist on desktop. @ericstj/@terrasjobst do we want to block such cases?

**Pull request merged**